core/arch/arm/plat-imx/imx.h

```diff
@@ -33,5 +33,10 @@ uint32_t imx_soc_type(void);
 void imx_gpcv2_set_core1_pdn_by_software(void);
 void imx_gpcv2_set_core1_pup_by_software(void);
 void imx_gpcv2_set_core_pgc(bool enable, uint32_t offset);
+
+#if !(defined(CFG_MX6Q) || defined(CFG_MX6D) || defined(CFG_TZC380))
+void imx_configure_tzasc(void) {}
```

Comment:
`static inline void` please.

Minor: it's usually written the other way around, without a negation (if defined() ... else ... endif)

Reply:
I also forgot the `#` for `else`. Fixed with a new Review commit and corrected logic.